src/app/layout/components/ActionsMenu.tsx

```diff
@@ -0,0 +1,47 @@
+
+import React from 'react';
+import { Button } from "@/components/ui/button"
+import { DotsHorizontalIcon } from '@radix-ui/react-icons';
+import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from '@radix-ui/react-dropdown-menu';
+
+interface Action {
```

Comment:
I think it'd be great if we pass `id` prop as well. Could be used later for identification.

Reply:
Okay